lib/sqlalchemy/engine/result.py

```diff
@@ -391,7 +398,7 @@ def _reduce(self, keys: Sequence[Any]) -> ResultMetaData:
 
 def result_tuple(
     fields: Sequence[str], extra: Optional[Any] = None
-) -> Callable[[Iterable[Any]], Row[Any]]:
+) -> Callable[[Iterable[Any]], Row[Unpack[_RawRowType]]]:
```

Comment:
**Michael Bayer** (zzzeek) wrote:

_RawRowType should be TupleAny

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/4986